client/src/docs-ui/page/page.tsx

```diff
@@ -200,15 +200,16 @@ export class DocsPage {
       }, []);
     this.blendUniversalNav = currentRoute === "/";
 
-    track({
-      type: AnalyticsEventType.PAGE_VISIT,
-      attributes: {
-        url: currentRoute,
-        previousUrl: this.previousPathname,
-        referrer: document.referrer,
-      },
-    });
-    trackPageVisit();
+    if (Build.isBrowser) {
```

Comment:
Do you need the `location.pathname === "/"` as well here?

Reply:
I don't understand why we would?  The page navigation event should fire whenever we switch pages, unless we're prerendering.